src/pages/MediaManip/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import styles from './DropDown.module.scss'
+
+function DropDown({type,onChange,value}) {
+
+    const dropDownItems = ['PNG','JPG','WEBP','JIFF']
+
+    return (
+        <select value={value} name="dropdown" className={styles.dropDown} onChange={(e) => onChange(e.target.value)}>
+            <option value={type}>{type}</option>
+            {dropDownItems.map(item => <option value={item}>{item}</option>)}
```

Comment:
Pass a `key` prop while mapping through the options array. Otherwise, React will throw an unwanted warnings in the console.